src/Framework/TelemetryEventArgs.cs

```diff
@@ -6,8 +6,6 @@
 using System.IO;
 using Microsoft.Build.Shared;
 
-#nullable disable
```

Comment:
Possibly controversial, but since the lack of nullability enforcement contributed to this bug I felt it was ok to onboard this type to it as part of this fix.

Reply:
I find that justification pretty convincing.